src/view/InventoryView.java

```diff
@@ -128,6 +118,11 @@ public void keyPressed(KeyEvent e) {
             public void keyReleased(KeyEvent e) {
             }
         });
+
+        this.setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        this.add(title);
+        this.add(itemInfo);
+        this.add(inventoryInfo);
```

Comment:
May have missed adding in "button" panel?

Reply:
added, thanks